babe

```diff
@@ -447,6 +447,8 @@ def decommission(mesosMasterEndpoint, hostname):
     tasks_on_slave = sorted([task for task in r.json()['tasks'] if task['slaveId'] == slaveId])
     candidate_tasks = sorted([getTaskInstance(task, lambda l: len(l) > 1) for task in tasks_on_slave])
     skipped_tasks = [getTaskInstance(task, lambda l: len(l) == 1) for task in tasks_on_slave if task not in candidate_tasks]
+    if decommissionSingleInstanceApps:
+        candidate_tasks = candidate_tasks + skipped_tasks
```

Comment:
Simpler thing would be to conditionally populate candidate_tasks` and `skipped_tasks` if flag is not set etc..?

Reply:
updated